truelearn/learning/_novelty_classifier.py

```diff
@@ -29,6 +27,40 @@ class NoveltyClassifier(InterestNoveltyKnowledgeBaseClassifier):
     is high, it means that neither side can easily win the other.
     Thus, a high quality game means that learners are likely to engage with
     learnable units based on our assumption.
+
+    Examples:
+        >>> from truelearn.learning import NoveltyClassifier
+        >>> from truelearn.models import EventModel, Knowledge, KnowledgeComponent
+        >>> novelty_classifier = NoveltyClassifier()
+        >>> novelty_classifier
+        NoveltyClassifier()
+        >>> # prepare event model
+        >>> knowledges = [
+        ...     Knowledge({1: KnowledgeComponent(mean=0.57, variance=1e-9)}),
+        ...     Knowledge({
+        ...         2: KnowledgeComponent(mean=0.37, variance=1e-9),
+        ...         3: KnowledgeComponent(mean=0.41, variance=1e-9),
+        ...     }),
+        ...     Knowledge({
+        ...         1: KnowledgeComponent(mean=0.24, variance=1e-9),
+        ...         3: KnowledgeComponent(mean=0.67, variance=1e-9),
+        ...     }),
+        ... ]
+        >>> events = [EventModel(knowledge) for knowledge in knowledges]
```

Comment:
unresolved reference knowledge in EventModel(knowledge)

Reply:
That's weird. Which tool do you run to get this output? This should be a doctest.